Split/Storage/Global/GlobalSecureStorage.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  GlobalSecureStorage.swift
+//  Split
+//
+//  Created by Javier L. Avrudsky on 03/29/2021.
+//  Copyright © 2021 Split. All rights reserved.
+//
+import Foundation
+
+class GlobalSecureStorage: KeyValueStorage {
```

Comment:
Javi, consider if this class deserves its own UTs

Reply:
Thanks doc!